k8s/base/kustomization.yaml

```diff
@@ -18,3 +18,6 @@ configMapGenerator:
   - name: openshift-quota-definition
     files:
       - quotas.json
+  - name: openshift-limit-definition
```

Comment:
I don't think I would make this a separate ConfigMap. I would suggest just adding `limits.json` to the exist CM:

```
configMapGenerator:
  - name: openshift-quota-definition
    files:
      - quotas.json
      - limits.json
```

...and then update the mounts accordingly.